static/app/views/performance/traceDetails/transactionBar.tsx

```diff
@@ -75,6 +75,17 @@ class TransactionBar extends Component<Props, State> {
     showDetail: false,
   };
 
+  componentDidMount() {
+    const {location, transaction} = this.props;
+
+    if (
+      'event_id' in transaction &&
+      transactionTargetHash(transaction.event_id) === location.hash
+    ) {
+      this.scrollIntoView();
+    }
+  }
+
```

Comment:
Had to put this lifecycle method here so that anchor links could work in the transaction trace view